homeassistant/components/fortios/device_tracker.py

```diff
@@ -47,49 +48,44 @@ def get_scanner(hass, config):
         return None
 
     status_json = fgt.monitor("system/status", "")
-    fos_major_version = int(status_json["version"][1])
 
-    if fos_major_version < 6 or fos_major_version > 7:
+    current_version = AwesomeVersion(status_json["version"][1:])
```

Comment:
Why is this sliced? what does `status_json["version"]` return?

Reply:
the first character is 'v'
e.g. "v6.4.3"

Reply:
Then you can drop the slicing,  `AwesomeVersion` handles that :)

Reply:
```suggestion
    current_version = AwesomeVersion(status_json["version"])
```